src/DayColumn.js

```diff
@@ -77,6 +77,34 @@ class DayColumn extends React.Component {
     this.slotMetrics = this.slotMetrics.update(nextProps)
   }
 
+  renderSelection() {
+    const { selecting } = this.state
+    if (selecting) {
+      let { top, height, startDate, endDate, noMovementYet } = this.state
+      const { localizer, components } = this.props
+      let selectDates = { start: startDate, end: endDate }
+      let selectionElement
+      if (
+        noMovementYet &&
+        components &&
+        components.beforeSelectIndicatorComponent != null
+      ) {
+        const SelectIndicator = components.beforeSelectIndicatorComponent
```

Comment:
sorry so delayed here :(

Can we just skip the component and render something like your screenshot as the default? Then folks can customize it via css instead

Reply:
Sure we can do that, I'll revise my PR when I find some free time and post back.

Reply:
https://github.com/intljusticemission/react-big-calendar/pull/1018 is ready